lib/internal/util/inspect.js

```diff
@@ -1022,6 +1027,11 @@ function formatRaw(ctx, value, recurseTimes, typedArray) {
       if (keys.length === 0 && protoProps === undefined) {
         return base;
       }
+    } else if (isURL(value) && !(recurseTimes > ctx.depth && ctx.depth !== null)) {
+      base = value.href;
+      if (keys.length === 0 && protoProps === undefined) {
+        return base;
```

Comment:
I think it would be valuable to see the object structure as before. With this change, it almost looks like a string without providing the information about being an URL object.
Overall, I am not fond of special handling URL during inspection, especially as our detection is only a heuristic that can definitely trigger for lots of other objects (`Boolean(self?.href && self.protocol && self.auth === undefined && self.path === undefined)`). We could see if we want to change the implementation in URL itself that no special handling would be needed.

Reply:
@BridgeAR Hi.
I thought it would be a simple solution to using `instanceof`, but I found that `Using a symbol or instanceof would not be able to recognize URL objects coming from other implementations (e.g. in Electron) in the annotation`

Could you tell me in what direction you think the implementation of the URL object should be changed?